docs/guides/js-waku/use-waku-react.md

```diff
@@ -0,0 +1,356 @@
+---
+title: "Build React DApps Using @waku/react"
+---
+
+The [@waku/react](https://www.npmjs.com/package/@waku/react) package provides components and UI adapters to integrate `js-waku` into React applications effortlessly. This guide provides detailed steps for using `@waku/react` in your project.
+
+## Install the Dependencies
+
+First, set up a project using any [production-grade React framework](https://react.dev/learn/start-a-new-react-project) or use an existing React application. For this guide, we will create a boilerplate using [Create React App (CRA)](https://create-react-app.dev/docs/getting-started):
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+<Tabs groupId="package-manager">
+<TabItem value="npm" label="npm">
+
+```shell
+npx create-react-app [PROJECT DIRECTORY]
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```shell
+yarn create react-app [PROJECT DIRECTORY]
+```
+
+</TabItem>
+</Tabs>
+
+Next, install the `@waku/react` package using your preferred package manager:
+
+<Tabs groupId="package-manager">
+<TabItem value="npm" label="npm">
+
+```shell
+npm install @waku/react @waku/sdk
```

Comment:
@weboko What do you think about making it so that you only need to install `@waku/react`?

Reply:
I only used the `utf8ToBytes` function from `@waku/sdk`. I can remove the explicit installation as npm/yarn will install it alongside `@waku/react`

Reply:
Nah it's best to explicitely install what you use, some checker will send an error if you don't

Reply:
> @weboko What do you think about making it so that you only need to install `@waku/react`?

this would be ideal, but to make it "right" I think there should be a set of abstractions for `React` ready
meaning I would avoid re-exporting everything from `@waku/sdk` through `@waku/react`